config.py

```diff
@@ -1,3 +1,24 @@
+import logging
+
+FORMAT = '%(asctime)s - %(name)-s - %(levelname)-s - %(message)s'
+LEVEL = logging.DEBUG
+logging.basicConfig(format=FORMAT, level=LEVEL)
+log = logging.getLogger(__name__)
+
+def logger(fn):
+    from functools import wraps
+    import inspect
+    @wraps(fn)
+    def wrapper(*args, **kwargs):
+        log = logging.getLogger(fn.__name__)
+        log.info('About to run %s' % fn.__name__)
+
+        out = apply(fn, args, kwargs)
+
+        log.info('Done running %s' % fn.__name__)
+        return out
+    return wrapper
+
```

Comment:
Add these to a new file `logger_config` and import that in the code
Also, remove extra whitespace

Reply:
Done 👍